src/main/java/frc/robot/subsystems/arm/Arm.java

```diff
@@ -0,0 +1,362 @@
+  new ProfiledPIDController(
+      ArmCal.ARM_P,
+      ArmCal.ARM_I,
+      ArmCal.ARM_D,
+      new TrapezoidProfile.Constraints(
+          ArmCal.ARM_MAX_VELOCITY_DEG_PER_SECOND,
+          ArmCal.ARM_MAX_ACCELERATION_DEG_PER_SECOND_SQUARED));
+
+  public Arm(ScoringLocationUtil scoreLoc) {
+
+    armPositionMap = new TreeMap<ArmPosition, Double>();
+    armPositionMap.put(ArmPosition.STARTING, ArmCal.ARM_START_POSITION_DEG);
+    armPositionMap.put(ArmPosition.INTAKE, ArmCal.ARM_INTAKE_POSITION_DEG);
+    armPositionMap.put(ArmPosition.SCORE_LOW, ArmCal.ARM_LOW_POSITION_DEG);
+    armPositionMap.put(ArmPosition.SCORE_MID_HIGH, ArmCal.ARM_HIGH_MID_POSITION_DEG);
+    armPositionMap.put(
+        ArmPosition.AVOID_LIMELIGHT, ArmCal.ARM_AVOID_LIMELIGHT_POSITION_DEG);
+
+  this.scoreLoc = scoreLoc;
+  }
+
+  public void initialize() {}
+
+  @Override
+  // This method will be called once per scheduler run
+  public void periodic() {
+    // Check if position has updated
+    if (atPosition(ArmPosition.STARTING)) {
+      this.latestPosition = ArmPosition.STARTING;
+    } else if (atPosition(desiredPosition)) {
+      this.latestPosition = desiredPosition;
+    }
+
+    controlPosition(desiredPosition);
+    armAbsoluteEncoderChecker.addReading(armAbsoluteEncoder.getPosition());
+  }
+
+  /** Sets the desired position */
+  public void setDesiredPosition(ArmPosition pos) {
+    desiredPosition = pos;
+  }
+
+  public void startScore() {
+    scoringInProgress = true;
+    if (scoreLoc.getScoreHeight() == ScoreHeight.HIGH || scoreLoc.getScoreHeight() == ScoreHeight.MID) {
+      setDesiredPosition(ArmPosition.SCORE_MID_HIGH);
+    } else if (scoreLoc.getScoreHeight() == ScoreHeight.LOW) {
+        setDesiredPosition(ArmPosition.SCORE_LOW);
+    }
+  }
+
+  public void deployArmLessFar() {
+    Double curAngle = armPositionMap.get(desiredPosition);
+    Double newAngle = curAngle - 0.5;
+    armPositionMap.replace(desiredPosition, newAngle);
+
+    new PrintCommand("Latest angle for " + desiredPosition + ": " + newAngle);
+
+    armController.setGoal(newAngle);
+  }
+
+  public void deployArmFurther() {
+    Double curAngle = armPositionMap.get(desiredPosition);
+    Double newAngle = curAngle + 0.5;
+    armPositionMap.replace(desiredPosition, newAngle);
+
+    new PrintCommand("Latest angle for " + desiredPosition + ": " + newAngle);
+
+    armController.setGoal(newAngle);
+  }
+
+  /** True if the arm is at the current value of this.desiredPosition */
+  private boolean atDesiredArmPosition() {
+    double armMarginDegrees =
+        desiredPosition == ArmPosition.STARTING
+            ? ArmCal.ARM_START_MARGIN_DEGREES
+            : ArmCal.ARM_MARGIN_DEGREES;
+    double armPositionToCheckDegrees = armPositionMap.get(desiredPosition);
+    double armPositionDegrees = armEncoder.getPosition();
+    if (Math.abs(armPositionDegrees - armPositionToCheckDegrees) > armMarginDegrees) {
+      return false;
+    }
+    return true;
```

Comment:
This can be condensed, you can just return the output of the boolean instead of needing an if statement.